RFC/0000-template.md

```diff
@@ -0,0 +1,63 @@
+# RFC 0000: Request for Change Template
+
+<!-- RFC number will be assigned by an editor. When opening a pull request to
+submit your RFC, please use an abbreviated title in the filename,
+`0000-draft-title-abbrev.md`. -->
+
+- Start Date: (format: YYYY-MM-DD)
+- Related Issues:
+  - (add links here)
+
+# Summary
+
+<!--One paragraph explanation of the RFC.-->
+This is the suggested template for new RFCs.
+
+# Motivation
+
+AKA Problem Statement
+
+Clearly explain why the existing protocol specification is inadequate
+to address the problem that the RFC solves.
+
+# Detailed design
+
+AKA Solution Proposal
+
+Describe the proposed solution and list all changes made to the specs repository.
+
+The resulting specification should be detailed enough to allow competing,
+interoperable implementations.
+
+## Test fixtures
+
+List relevant CIDs. Describe how implementations can use them to determine
+specification compliance.
+
+## Design rationale
+
+The rationale fleshes out the specification by describing what motivated
+the design and why particular design decisions were made.
+
+Provide evidence of consensus within the community,
```

Comment:
instead of "consensus" here (which could be interpreted too strictly) should we inject a bit of the spirit of IPFS specs and say "evidence of rough consensus and working code"?

Reply:
```suggestion
Provide evidence of rough consensus and working code within the community,
```